sound/soc/intel/boards/sof_maxim_common.c

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: GPL-2.0
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+#include <sound/pcm.h>
+#include <sound/soc.h>
+#include "sof_maxim_common.h"
+
+static const struct snd_soc_dapm_route max_98373_dapm_routes[] = {
```

Comment:
strictly speaking for this you have to `#include <sound/soc-dapm.h>`

Reply:
Done. I have updated patchset.
@lyakh : please review.

Reply:
no, sorry, that isn't what I meant. The standard approach in the kernel is to include all headers where they are needed.

Reply:
I have included all snd header files need. Please review now. Thanks!